src/utils/index.js

```diff
@@ -61,7 +61,7 @@ function checkAppPath(appPath) {
 
 function getAppTemplateConfig(templatePath, { loadFileFn = require } = {}) {
   try {
-    return loadFileFn(`${templatePath}/.template.js`);
+    return loadFileFn(path.resolve(`${templatePath}/.template.js`));
```

Comment:
probably best to use path.join, so it works on Windows?

Reply:
something that should be added everywhere though, not just here. If we decide to properly support windows, we can add CI for that etc.